modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -328,10 +332,12 @@ class SeqexecEngine(httpClient: Client[IO], settings: SeqexecEngine.Settings, sm
 
   def startQueue(q: EventQueue, qid: QueueId, observer: Observer, user: UserDetails, clientId: ClientId): IO[Either[SeqexecFailure, Unit]] = q.enqueue1(
     Event.modifyState[executeEngine.ConcreteTypes](executeEngine.get.flatMap{ st => {
-      queueO(qid).getOption(st).map {
+      queueO(qid).getOption(st).filter(!_.queue.isEmpty).map {
         _.status(st) match {
-          case BatchExecState.Idle     => ((EngineState.queues ^|-? index(qid) ^|-> ExecutionQueue.cmdState).set(BatchCommandState.Run(observer, user, clientId)) >>> {(_, ())}).toHandle *> runQueue(qid, observer, user, clientId)
-          case BatchExecState.Stopping => ((EngineState.queues ^|-? index(qid) ^|-> ExecutionQueue.cmdState).set(BatchCommandState.Run(observer, user, clientId)) >>> {(_, ())}).toHandle
+          case BatchExecState.Idle |
+               BatchExecState.Stopping => ((EngineState.queues ^|-? index(qid) ^|-> ExecutionQueue.cmdState)
+            .set(BatchCommandState.Run(observer, user, clientId)) >>> {(_, ())}).toHandle *>
+            runQueue(qid, observer, user, clientId)
```

Comment:
`runQueue` must be called when state is Stopping, because some sequence could have already been stopped